patches/PRODSECBUG-2432__admin_path_disclosure_bug__2.3.0.patch

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Format + Change first line to be same as our other patches with parameters. Also drop index